pkg/controller/persistentvolume/types.go

```diff
@@ -25,6 +25,17 @@ import (
 	"k8s.io/kubernetes/pkg/client/cache"
 )
 
+const (
+	// A PVClaim containing this annotation requests a quality of service tier.
+	// The value of this field is arbitrary and known by the user and administrator.
+	// This is an experimental feature and likely to change in the future.
+	qualityOfServiceRequestKey = "experimental/k8s.io/persistentVolumeProvisioning-qos"
```

Comment:
Since it's not yet used anywhere, let's remove this (it can be re-added along with code using it).